scripts/build_go_slim.sh

```diff
@@ -92,3 +104,12 @@ for f in ./coder-slim_*; do
 	dest="$dest_dir/$hyphenated"
 	cp "$f" "$dest"
 done
+
+if [[ -n $compress ]]; then
+	(
+		cd "$dest_dir"
+		tar cf coder.tar coder-*
+		rm coder-*
+		zstd --ultra --long -"${compress}" --rm --no-progress coder.tar -o coder.tar.zst
+	)
```

Comment:
```suggestion
	cd "$dest_dir"
	tar cf coder.tar coder-*
	rm coder-*
	zstd --ultra --long -"${compress}" --rm --no-progress coder.tar -o coder.tar.zst
```

Reply:
I used `( )` here to avoid the side-effect of changing directory, in case something is appended to this script. Is that unwanted / unnecessary?

Reply:
In that case use `pushd` and `popd` instead